### PR TITLE
Reenable CI workflow

### PR DESCRIPTION
**Pull request by yolocs**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -90,279 +90,272 @@ jobs:
...
+          workload_identity_provider: '${{ env.WIF_PROVIDER }}'
+          service_account: '${{ env.WIF_SERVICE_ACCOUNT }}'
+          token_format: 'access_token'
+      # TODO: Workload Identity doesn't work with GCS Provider.
+      # https://github.com/google-github-actions/auth/issues/89
+      - name: 'Workaround Terraform issue'
+        shell: 'bash'
+        run: |-
+          echo "GOOGLE_OAUTH_ACCESS_TOKEN=${{ steps.auth.outputs.access_token }}" >> $GITHUB_ENV
+      - uses: hashicorp/setup-terraform@633666f66e0061ca3b725c73b2ec20cd13a8fdd1 # ratchet:hashicorp/setup-terraform@v2
```
Comment (verbanicm) [inline .github/workflows/ci.yml:302]: quote yaml

**Review by verbanicm — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -15,11 +15,11 @@ on:
 env:
   DOCKER_REGISTRY: 'us-docker.pkg.dev'
   DOCKER_TAG: '${{ github.sha }}'
-  DOCKER_REPO: 'us-docker.pkg.dev/lumberjack-dev-infra/images'
-  WIF_PROVIDER: 'projects/638387980668/locations/global/workloadIdentityPools/github-pool/providers/github-provider'
-  WIF_SERVICE_ACCOUNT: 'gh-access-sa@lumberjack-dev-infra.iam.gserviceaccount.com'
-  SERVER_PROJECT_ID: 'github-ci-server'
-  CLIENT_PROJECT_ID: 'github-ci-app-0'
+  DOCKER_REPO: 'us-docker.pkg.dev/gh-lumberjack-ci-i-bbdb31/ci-images'
+  WIF_PROVIDER: 'https://iam.googleapis.com/projects/201020025366/locations/global/workloadIdentityPools/gh-automation-i-c38297/providers/gh-automation-i-c38297'
```
Comment (verbanicm) [inline .github/workflows/ci.yml:19]: Omit the http address....

Comment (yolocs) [reply .github/workflows/ci.yml]: done

File: .github/workflows/ci.yml (hunk 1 above)
Comment (yolocs) [reply .github/workflows/ci.yml:302]: time to fix tech debt 😭

**Review by sethvargo — APPROVED** [review]

**Pull request merged**